### ISSUE TITLE
Make HTML structure valid

### ISSUE THREAD

**Issue by maxliqd** [liqd/digitalstrategie#607]
**Essential Information** 
**URL:** gemeinsamdigital.berlin.de
**device & browser:**
**operating system (incl. version):**
**behaviour:**
The following errors were found:

General:

- [x] Meta tag with ""charset"" although an ""http-equiv"" is already set.

- [x] Overlays (accessibility, search, menu): Missing attribute ""role"" on DIVs with ""aria-modal""

- [x] Incorrect use of the attribute ""itemprop""

https://gemeinsamdigital.berlin.de/de/

- [x] ""alt"" and ""loading"" attribute on DIV elements not allowed here

https://gemeinsamdigital.berlin.de/de/das-ist-gemeinsam-digital-berlin/

- [x] Duplicate ID ""layout-grid"".

https://gemeinsamdigital.berlin.de/de/strategie/warum-gdb/

~~- [ ] P tags nested inside each other.~~ can't find them?

https://gemeinsamdigital.berlin.de/de/kontaktformular/

- [x] A non-existent attribute ""combined_answer_id"" is used without prefixing it with ""data-"", for example.
- [x] Label for ""id_captcha"" points to a hidden input."

**expected behaviour:** There should be no HTML errors on the page that may confuse a screen reader or other assistive tools.

**Additional information if relevant**
**user role:**
**important screensize:**
**Comment/Question:**

**Screenshot if design related**:
Since issues are public, make sure to not display personal or secret data.

**Comment by philli-m** [comment]
@fuzzylogic2000 @Rineee the django form field makes it a bit difficult to change the id the label is for, just wondering if eitehr of you can take a look? I can also try next week!

**Comment by maxliqd** [comment]
@philli-m Ok cool. I trust you, did not double check though and the audit was not more specific on this matter.

**Comment by philli-m** [comment]
@maxliqd I gave it a quick check but i'll do a proper run through with a SR on the filters and the form submission after related issues are done
